validator/src/test/java/io/avaje/validation/core/AllSortsBeanValidationAdapter.java

```diff
@@ -0,0 +1,52 @@
+package io.avaje.validation.core;
+
+import io.avaje.validation.adapter.ValidationAdapter;
+import io.avaje.validation.adapter.ValidationContext;
+import io.avaje.validation.adapter.ValidationRequest;
+import jakarta.validation.constraints.*;
+
+import java.util.Collections;
+import java.util.Map;
+
+public final class AllSortsBeanValidationAdapter implements ValidationAdapter<AllSortsBean> {
```

Comment:
so gotta ask, since we have the generator up and running, do we really need to write these by hand still?